Library/Homebrew/extend/os/mac/hardware/cpu.rb

```diff
@@ -140,6 +156,7 @@ def arm_family
         end
       end
 
+      sig { params(_family: T.untyped, _cpu_model: T.untyped).returns(Symbol) }
       def intel_family(_family = nil, _cpu_model = nil)
```

Comment:
```suggestion
      sig { params(_family: Integer, _cpu_model: Integer).returns(Symbol) }
      def intel_family(_family = T.unsafe(nil), _cpu_model = T.unsafe(nil))
```
or
```suggestion
      sig { params(_family: T.nilable(Integer), _cpu_model: T.nilable(Integer)).returns(Symbol) }
      def intel_family(_family = nil, _cpu_model = nil)
```
to match definitions elsewhere